Controllers/PlayerTableViewCell.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  PlayerTableViewCell.swift
+//  NBA news app
+//
+//  Created by Klaidas Sinkevičius on 2022-10-14.
+//
+
+import UIKit
+
+class PlayerTableViewCell: UITableViewCell {
+    
+    @IBOutlet var playerInfo: UILabel!
+    @IBOutlet var playerIcon: UIImageView!
+    
+    var player: PlayerClass?
+    var teamID: String?
+    
+    func setupPlayer(){
+        guard let player = player else {
+            return
+        }
+        
+        guard player.teamID == teamID else {
+            return
+        }
+        
+        if player.image != "" {
+            playerIcon.load(stringURL: player.image ?? "")
```

Comment:
```suggestion
        if let playerImage = player.image, playerImage != "" {
            playerIcon.load(stringURL: playerImage)
```
It is best to avoid `??`s if possible.